### PR TITLE
OpenMC: add v0.13.2

### PR DESCRIPTION
**Pull request by paulromano**
This PR adds version 0.13.2 for packages openmc and py-openmc.

### CODE REVIEW

**Review by tldahlgren — DISMISSED** [review]
Confirmed both new commits.

**Comment by tldahlgren** [comment]
Odd precheck/validate failures:

```
...
WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7ffbe8c8[19](https://github.com/spack/spack/actions/runs/3465196156/jobs/5787645006#step:4:20)10>: Failed to establish a new connection: [Errno 101] Network is unreachable')': /packages/09/bd/2410905c76ee14c62baf69e3f4aa780226c1bbfc9485731ad018e35b0cb5/pip-22.3.1-py3-none-any.whl
19
ERROR: Could not install packages due to an OSError: HTTPSConnectionPool(host='files.pythonhosted.org', port=443): Max retries exceeded with url: /packages/09/bd/2410905c76ee14c62baf69e3f4aa780226c1bbfc9485731ad018e35b0cb5/pip-22.3.1-py3-none-any.whl (Caused by NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7ffbe8c81fd0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
```

Will close and re-open the PR.

**Comment by tldahlgren** [comment]
@adamjstewart Did you want to comment on the Python package included here?

File: var/spack/repos/builtin/packages/py-openmc/package.py [hunk 1]
```diff
@@ -33,7 +34,17 @@ class PyOpenmc(PythonPackage):
...
+        "0.13.1",
+        "0.13.0",
+        "0.12.2",
+        "0.12.1",
+        "0.12.0",
+        "0.11.0",
+    ]:
         depends_on(
             "openmc+mpi@{0}".format(ver), when="@{0}+mpi".format(ver), type=("build", "run")
         )
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-openmc/package.py:50]: Dependencies need to be updated. Not sure when these things changed, but the latest version requires:

* `python@3.7:`
* `numpy@1.9:1.21`

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-openmc/package.py (hunk 1 above)
Comment (paulromano) [reply var/spack/repos/builtin/packages/py-openmc/package.py:50]: Good point -- I've updated the dependency for Python. For numpy, what's currently listed (`@1.9:`) should still apply.

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-openmc/package.py:50]: According to https://github.com/openmc-dev/openmc/blob/v0.13.2/pyproject.toml, numpy 1.22 is not yet supported.

Comment (paulromano) [reply var/spack/repos/builtin/packages/py-openmc/package.py:50]: Ah yes, you're right. I've just updated the numpy dependency.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**